src/mixins/editMixin.js

```diff
@@ -26,14 +26,22 @@ export default {
     hasChanged () {
       return !this.isNew && !deepEqual(this.value, this.edit)
     },
+    today () {
```

Comment:
Builtin classes (e.g. `Date`) are not reactive, so this will never update.

You could either just initialize it in the `data()` function (it doesn't really _need_ to be reactive).

Or, if you want a reactive value, you can get it using:

```js
import dateFnsHelper from '@/services/dateFnsHelper'

// ... stuff

computed: {
  now () {
    return dateFnsHelper.now // <--- this value is reactive!
  }
}
```

Reply:
Fix pushed

Reply:
Ah, I just noticed this is in the `editMixin`. It would be more appropriate in the `PickupEdit` component as the past date only relates to that, not editing in general.

Also, when you move it, maybe rename the computed property to `now` so it's consistent with the name inside the helper. Thanks!